webrtc.html

```diff
@@ -450,6 +451,15 @@
                 <code><a data-link-for=
                 "RTCPeerConnection">localDescription</a></code> attribute.</p>
               </dd>
+              <dt><dfn><code>reofferCodecs</code></dfn> of type <span class=
+              "idlMemberType"><a>boolean</a></span>, defaulting to
+              <code>false</code></dt>
+              <dd>
+                <p>When the value of this dictionary member is true,
+                the generated description will include codecs that
+                were excluded by the current remote description, as
```

Comment:
Technically, the codecs are restricted by the last applied answer, not the current remote description.